mux/broadcaster_test.go

```diff
@@ -0,0 +1,73 @@
+package mux
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+const numberOfMessagesSent = 20
```

Comment:
If we want to add tests in this file, we're going to have some problems because we have global values here.
I think we would be better to move the declaration of `numberOfMessageSent`, `blockingClientChan` and `finished`  inside TestBackpressureOnBroadcaster, and pass `finished` to `consume` as argument

Reply:
yep